### ISSUE TITLE
Problem with textContentType

### ISSUE THREAD

**Issue by alexanderkhitev** [marmelroy/PhoneNumberKit#215]
I set `phoneNumberTextField.textContentType = UITextContentType.telephoneNumber `and when I try to insert a phone number from the prompt, nothing happens. I guess because the phone number has brackets (***) *** - ** - **. How can I fix this and what can I do to make it work with `PhoneNumberTextField`?

**Comment by wzs** [comment]
It seems to be a problem with UIKit - after setting `textContentTyp`, when you tap a "predicted" text on the top of keyboard, the `replacementString` argument in delegate method `textField:shouldChangeCharactersInRange:replacementString:` is always set to just a single space - therefore `PhoneNumberKit` doesn't work.

related: [rdar://33634848](http://www.openradar.me/33634848)

**Comment by rafaelreis-hotmart** [comment]
Hey guys, the extra space is because `smartInsertDeleteType` property is saying iOS to do that.
To change that behavior setup as follow:
```
        if #available(iOS 11.0, *) {
            phoneNumberTextField.smartInsertDeleteType = .no
        }
```
